### PR TITLE
[VIRTS-1879] Better docs structure

### PR DESCRIPTION
**Pull request by b17zr**
## Description

- Added Basic Usage page and move most of the former "Learning the Terminology" content there
- Rewrote "Learning the Terminology" page
- Removed CALDERA 2.0 information
- Added "Troubleshooting" page, from the "Common problems" (I felt like this name implied something..) page

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [?] This change requires a documentation update

## How Has This Been Tested?

Regenerated docs with sphinx.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [?] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by b17zr** [comment]
Also updated the server configuration page to include startup parameters (VIRTS-1831) and a documented config file.

File: sphinx-docs/Basic-Usage.md
Comment (bworrell) [inline sphinx-docs/Basic-Usage.md:10]: The `button5 n` at the end of the line looks out of place. Is that intentional?

File: sphinx-docs/Basic-Usage.md
Comment (bworrell) [inline sphinx-docs/Basic-Usage.md:309]: `PLugin`

File: sphinx-docs/Basic-Usage.md
Comment (bworrell) [inline sphinx-docs/Basic-Usage.md:109]: `an` should be `An`

**Review by bworrell — CHANGES_REQUESTED** [review]
I pulled this down and ran `make clean html` in the sphinx docs folder and it built/rendered fine (loaded it up in my browser). I didn't go through this too carefully but only noticed a couple of small typos. Overall this looks like an improvement in the structure of the documentation.

**Comment by b17zr** [comment]
https://github.com/mitre/fieldmanual/pull/63 will conflict, but I've included the DNS config information in Server-Configuration.md.

**Review by bworrell — DISMISSED** [review]
LGTM! I pulled it down and rebuilt the docs locally and it all loaded up correctly for me.

File: sphinx-docs/Server-Configuration.md
Comment (bworrell) [inline sphinx-docs/Server-Configuration.md:25]: It looks like some of the DNS configuration info may have gotten lost during the merge conflict resolution. Is it worth including some of the info from the original file? https://github.com/mitre/fieldmanual/pull/64/files#diff-8d8033dc12d18ad455b4300a0f7237f86700a4f8fd177e0cbd566b064272fc51L45-L46

Comment (b17zr) [reply sphinx-docs/Server-Configuration.md:25]: The only information I see missing is "The C2 server will respond with the NXDOMAIN response code for any other domain". It feels more appropriate to put this into a page or section dedicated to the different contact methods.

**Review by bworrell — APPROVED** [review]
LGTM after resolving merge conflicts

**Pull request merged**
